rocksdb_replicator/rocksdb_replicator.h

```diff
@@ -130,6 +130,7 @@ class RocksDBReplicator {
     std::shared_ptr<replicator::DbWrapper> db_wrapper_;
     folly::Executor* const executor_;
     const ReplicaRole role_;
+    const char* role_str_;
```

Comment:
I am not sure, whether this is good practice to include a string version to the class property. 
Since it does not really provide any other info to the class.

Reply:
it's pretty common to cache frequent accessed variables, rather than compute on the fly each time its accessed (in this case it's the admin endpoint). And the memory overload of caching a simple short string is very low.